agent-control/src/config_migrate/migration/defaults.rs

```diff
@@ -5,7 +5,10 @@ configs:
   -
     agent_type_fqn: newrelic/com.newrelic.infrastructure:0.1.0
     filesystem_mappings:
-      config_agent: /etc/newrelic-infra.yml
+      config_agent:
+        file_path: /etc/newrelic-infra.yml
+        overwrites: {}
+        deletions: []
```

Comment:
I'd like to check if I understood it right: This is the config used by the migration binary and we've added the empty `overwrites` and `deletion` to keep its behavior, right?

When deleting the binary, can we also get rid of this constant?

Reply:
This is used both by the binary and by the new code. Those fields are there to be able to be parsed by serde_yaml (some default could be added to avoid having the declare them empty but I didn't see the benefit).
The migrator needs a configuration like this in order to look for files and maps to convert, I would continue having a constant even if a struct with the defaults would also work.